src/core_plugins/kibana/public/dashboard/index.html

```diff
@@ -59,5 +59,13 @@
     <p>Click the <a class="btn btn-xs navbtn-inverse" ng-click="kbnTopNav.open('add'); toggleAddVisualization = !toggleAddVisualization" aria-label="Add visualization">Add</a> button in the menu bar above to add a visualization to the dashboard. <br/>If you haven't setup a visualization yet visit the <a href="#/visualize" title="Visualize">"Visualize"</a> tab to create your first visualization.</p>
   </div>
 
-  <dashboard-grid></dashboard-grid>
+  <dashboard-grid ng-if="expandedPanel === null"></dashboard-grid>
+
+  <dashboard-panel ng-if="expandedPanel !== null"
```

Comment:
It might be a nice, small improvement to encapsulate this logic inside of a scope method, e.g. `hasExpandedPanel()`. I think that will make the template a little more readable.